### PR TITLE
Read container output from temp file instead of Std Out

### LINKED ISSUE #107: use file path for program outputs

### PR DESCRIPTION
**Pull request by mikehostetler**
Closes #107 

** This code hasn't been tested due to challenges with running `runsc` localy **

This code does the following:

1. Creates a temporary file based upon the container path and container id - ensures that file exists on disk - ensures the file is blank
2. Redirects the Command to execute `runsc` to dump its output to this temporary file
3. Reads the contents of the temporary file and returns them instead
4. Deletes the file after it has been used up

### CODE REVIEW

**Comment by eureka-cpu** [comment]
Does this close #107 ?

**Comment by mikehostetler** [comment]
Yes - this would close #107

File: crates/compute/src/compute.rs [hunk 1]
```diff
@@ -351,6 +351,26 @@ impl OciManager {
...
+                tokio::fs::create_dir_all(parent).await?;
+            }
+
+            // Check if the file exists
+            if tokio::fs::metadata(&temp_file_path).await.is_ok() {
+                // If it exists, make sure it is empty
+                tokio::fs::write(&temp_file_path, b"").await?;
+            } else {
+                // If it doesn't exist, create it
+                let _ = tokio::fs::File::create(&temp_file_path).await?;
```
Comment (andrewvious) [inline crates/compute/src/compute.rs:368]: The first create file call using tokio can be removed, as it's handled with std in the following lines where it's being used for stdout redirection.

**Review by andrewvious — CHANGES_REQUESTED** [review]
Everything seems to working as intended, here is the logging info I was able to get during testing:
```lasr   | 2024-05-29T22:13:59.983Z ERROR [lasr_compute::compute] Calling: runsc --rootless --network=none run -bundle ./containers/bafyreid6i4ych4otqzurhamkh3ep6d43dsn2ovlncdw32upmkc6rlabbb4 bafyreid6i4ych4otqzurhamkh3ep6d43dsn2ovlncdw32upmkc6rlabbb4
lasr   | 2024-05-29T22:13:59.985Z ERROR [lasr_compute::compute] temp file path established for std::out "/tmp/lasr/bafyreid6i4ych4otqzurhamkh3ep6d43dsn2ovlncdw32upmkc6rlabbb4.out"
lasr   | 2024-05-29T22:13:59.985Z ERROR [lasr_compute::compute] temp dir created: "/tmp/lasr"
lasr   | 2024-05-29T22:13:59.985Z ERROR [lasr_compute::compute] temp file not found, creating with tokio call: "/tmp/lasr/bafyreid6i4ych4otqzurhamkh3ep6d43dsn2ovlncdw32upmkc6rlabbb4.out"
lasr   | 2024-05-29T22:13:59.985Z ERROR [lasr_compute::compute] temp file created again? : "/tmp/lasr/bafyreid6i4ych4otqzurhamkh3ep6d43dsn2ovlncdw32upmkc6rlabbb4.out"
lasr   | 2024-05-29T22:13:59.986Z ERROR [lasr_compute::compute] passing inputs to stdio: "{lots of account/metadata, removed for sake of readability}"
lasr   | 2024-05-29T22:14:00.455Z ERROR [lasr_compute::compute] result from container: bafyreid6i4ych4otqzurhamkh3ep6d43dsn2ovlncdw32upmkc6rlabbb4 = "{\"computeInputs\":{lots of account/metadata, removed for sake of readability}"
lasr   | 2024-05-29T22:14:00.456Z ERROR [lasr_compute::compute] results received, informing executor
lasr   | 2024-05-29T22:14:00.456Z ERROR [lasr_compute::compute] casted message to inform executor
lasr   | 2024-05-29T22:14:00.456Z ERROR [lasr_compute::compute] cleaning up temp file...

**Review by eureka-cpu — APPROVED** [review]

**Pull request merged**